packages/salesforcedx-utils-vscode/package.json

```diff
@@ -29,6 +29,9 @@
     "source-map-support": "^0.4.15",
     "typescript": "2.4.0"
   },
+  "engines": {
+    "vscode": "^1.15.0"
```

Comment:
@muenzpraeger - Can we peg this one at ^1.13 for now like the rest of the extensions (with the exception of the debugger that does need 1.15 features but is not shipping yet).

Also, the formatting seems a bit off. Did you apply the prettier formatter?

Reply:
@vazexqi I'm wondering where these formatting changes (see keys "categories" and "activationEvents" are coming from.

https://github.com/muenzpraeger/salesforcedx-vscode/commit/53761a4308edf25df231815cab8d90ea67e00ab0#diff-d79693d63f96a4709135983f4a6f9e12R24

Prettier 0.22.1 is installed and I haven't applied any custom formatting for it. Would be nice to know where the changes are coming from.

Reply:
This, annoyingly, comes from the publish process. See https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/scripts/publish.js#L51 

I don't know why when we use lerna to do the publishing, it cannot just reformat the lines (the dependencies) that it needs to update. Instead, it formats the file.

If someone could investigate why this comes to be, I would be very grateful. This is likely an issue with lerna itself.